### PR TITLE
[PDO] fix: Explicitly convert PDO queries to utf-8

### PR DESCRIPTION
**Pull request by John98Zakaria**
This explicitly converts queries to utf-8 to allow them to be sent over grpc

I am not sure what the consequences are for  Chinese or Arabic queries :sweat_smile: or any other non Latin alphabet language

Closes: https://github.com/open-telemetry/opentelemetry-php/issues/1381

### CODE REVIEW

**Comment by John98Zakaria** [comment]
I fixed the php-stan warning for my code. :pray:

File: composer.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
   "prefer-stable": true,
   "require": {
     "php": "^7.4 || ^8.0",
-    "ext-json": "*"
+    "ext-json": "*",
+    "ext-mbstring": "*"
```
[the remark this answers is not in the document]
Comment (ChrisLightfootWild) [reply composer.json:11]: Such changes should also be added to `src/Instrumentation/PDO/composer.json` 👍

[the remark this answers is not in the document]
Comment (John98Zakaria) [reply composer.json:11]: * I removed the requirement from the root composer json

* Added `symfony/polyfill-mbstring` as a requirement

* Added `ext-mbstring` the extension as a suggested

**Comment by John98Zakaria** [comment]
I added tests that check for utf-8 encoding

**Review by brettmc — APPROVED** [review]

**Pull request merged**